irohad/multi_sig_transactions/state/mst_state.hpp

```diff
@@ -144,7 +152,7 @@ namespace iroha {
 
     using InternalStateType =
         std::unordered_set<DataType, iroha::model::PointerTxHasher<DataType>,
-                           iroha::DereferenceEquals<DataType>>;
```

Comment:
If `DereferenceEquals` is not used anywhere now, it can be either removed, or a TODO with task id should be added.